File: id0.java
import java.io.*;
import java.nio.*;
import java.util.*;
import java.security.*;

class ID0 {
    public static void main(String[] args) {
        try {
            RandomAccessFile raf = new RandomAccessFile("movable.sed", "r");
            byte[] bytes = new byte[0x10];
            raf.seek(0x110);
            raf.read(bytes);
            MessageDigest digest = MessageDigest.getInstance("SHA-256");
            byte[] hash = digest.digest(bytes);
            byte[] tmp = new byte[4];
            
            for(int i = 0; i < 4; i++) {
                System.arraycopy(hash, i * 4, tmp, 0, 4);
                ByteBuffer buf = ByteBuffer.wrap(tmp);
                buf.order(ByteOrder.LITTLE_ENDIAN);
                System.out.printf("%08X", buf.getInt(0));
            }
        }
        catch(FileNotFoundException e) {e.printStackTrace();}
        catch(IOException e) {e.printStackTrace();}
        catch(NoSuchAlgorithmException e) {e.printStackTrace();}
    }
}

File: id0.pl
#!/usr/bin/perl

use Digest::SHA qw(sha256_hex);

my $buf;
open my $fh, "<", "movable.sed" or die $!;
binmode $fh;
seek $fh, 0x110, 0;
read $fh, $buf, 0x10;
close $fh;
my $hash = sha256_hex($buf);
my @array = $hash =~ /.{8}/g;
for(my $i = 0; $i < 4; $i++){
    print reverse(@array[$i] =~ /.{2}/g)
}